CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All of the packages in the `apollo-server` repo are released with the same versi
 
 ### vNEXT
 
-- Pass the response tollkit to the context function. [#1407](https://github.com/apollographql/apollo-server/pull/1407)
```

Comment:
Hey my `tollkit` was... you know, trying to generate revenue for this awesome open-source project or something :) Me no make text good.